lib/trim-emoji.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const emojiRegex = require('emoji-regex');
+
+module.exports = text => {
+	let match;
+	let regex = emojiRegex();
+	while ((match = regex.exec(text))) {
+		const emoji = match[0];
+		const codePoints = [...emoji].length;
+		if (match.index === 0) {
+			text = text.substr(codePoints).trim();
+			regex = emojiRegex();
+		} else if (match.index + codePoints === text.length) {
+			text = text.substr(0, match.index).trim();
+			regex = emojiRegex();
+		}
+	}
+
+	return text;
```

Comment:
Why not just `text.replace(emojiRegex, '')`?

Reply:
See my test cases. I want to trim emojis and spaces, but **NOT** to remove inner emojis and **NOT** to trim spaces if no emojis.

- t.deepEqual(trimEmoji('⭐ hello'), 'hello');
- t.deepEqual(trimEmoji(' hello '), ' hello ');
- t.deepEqual(trimEmoji('hello⭐hello'), 'hello⭐hello');